extensions/cl_ext_float_atomics.asciidoc

```diff
@@ -0,0 +1,641 @@
+C atomic_fetch_max_explicit(volatile __global A *object, M operand, memory_order order, memory_scope scope)
+
+// In addition to the requirements described in the OpenCL C 3.0 specification,
+// requires the __opencl_c_ext_fp16_local_atomic_min_max feature (for atomic_half),
+// requires the __opencl_c_ext_fp32_local_atomic_min_max feature (for atomic_float), or
+// requires the __opencl_c_ext_fp64_local_atomic_min_max feature (for atomic_double).
+C atomic_fetch_min(volatile __local A *object, M operand)
+C atomic_fetch_max(volatile __local A *object, M operand)
+C atomic_fetch_min_explicit(volatile __local A *object, M operand, memory_order order)
+C atomic_fetch_max_explicit(volatile __local A *object, M operand, memory_order order)
+C atomic_fetch_min_explicit(volatile __local A *object, M operand, memory_order order, memory_scope scope)
+C atomic_fetch_max_explicit(volatile __local A *object, M operand, memory_order order, memory_scope scope)
+
+// In addition to the requirements described in the OpenCL C 3.0 specification,
+// requires the __opencl_c_ext_fp16_global_atomic_min_mas feature
+// or the __opencl_c_ext_fp16_local_atomic_min_mas feature (for atomic_half),
+// requires the __opencl_c_ext_fp32_global_atomic_min_mas feature
+// or the __opencl_c_ext_fp32_local_atomic_min_mas feature (for atomic_float), or
+// requires the __opencl_c_ext_fp64_global_atomic_min_mas feature
+// or the __opencl_c_ext_fp64_local_atomic_min_mas feature (for atomic_double).
+C atomic_fetch_min(volatile A *object, M operand)
+C atomic_fetch_max(volatile A *object, M operand)
+C atomic_fetch_min_explicit(volatile A *object, M operand, memory_order order)
+C atomic_fetch_max_explicit(volatile A *object, M operand, memory_order order)
+C atomic_fetch_min_explicit(volatile A *object, M operand, memory_order order, memory_scope scope)
+C atomic_fetch_max_explicit(volatile A *object, M operand, memory_order order, memory_scope scope)
+----
+
+== Modifications to the OpenCL API Specification
+
+Add to Table 5 - OpenCL Device Queries in Section 4.2 - Querying Devices: ::
++
+--
+[caption="Table 5. "]
+.List of supported param_names by clGetDeviceInfo
+[width="100%",cols="3,2,5",options="header"]
+|====
+| Device Info | Return Type | Description
+| `CL_DEVICE_SINGLE_FP_ATOMIC_CAPABILITIES_EXT`
+  `CL_DEVICE_DOUBLE_FP_ATOMIC_CAPABILITIES_EXT`
+  `CL_DEVICE_HALF_FP_ATOMIC_CAPABILITIES_EXT`
+  | `cl_device_fp_atomic_capabilities`
+      | Describes the floating-point atomic operations supported by the device.
+        This is a bit-field that describes a combination of the following values:
+
+        `CL_DEVICE_GLOBAL_FP_ATOMIC_LOAD_STORE_EXT` - Can perform floating-point load, store, and exchange atomic operations in global memory. +
+        `CL_DEVICE_GLOBAL_FP_ATOMIC_ADD_EXT` - Can perform floating-point addition and subtraction atomic operations in global memory.
```

Comment:
It looks to me as though there is a missing "+" at the end of this line.

Reply:
Fixed - thanks!